plugin/evm/atomic/tx.go

```diff
@@ -288,6 +289,37 @@ func SortEVMInputsAndSigners(inputs []EVMInput, signers [][]*secp256k1.PrivateKe
 	sort.Sort(&innerSortInputsAndSigners{inputs: inputs, signers: signers})
 }
 
+// EffectiveGasPrice returns the price per gas that the transaction is paying
+// denominated in aAVAX/gas.
+//
+// The result is rounded down to the nearest aAVAX/gas.
+func EffectiveGasPrice(
+	tx UnsignedTx,
+	avaxAssetID ids.ID,
+	isApricotPhase5 bool,
+) (uint256.Int, error) {
+	gasUsed, err := tx.GasUsed(isApricotPhase5)
```

Comment:
[No action] Is there a TLDR on the historic context of why `isApricotPhase5` added a fixed gas cost to atomic txs? Purely for my own context.